synapse/lib/membrane.py

```diff
@@ -0,0 +1,28 @@
+import synapse.lib.auth as s_auth
+
+class Membrane:
```

Comment:
I think we should make the Membrane a ``EventBus`` so we can just fini() it and have that fini() remove the ``on('splice',...)`` handler later

Reply:
Do we want the Membrane instances to be adding/removing event handlers on the core's eventbus? One of the things Visi and I talked about earlier on is that Membrane wouldn't be coupled with Cortex. I guess Membrane could take a fini func and the core could pass in a fini func that would remove the event handler that it just added?